pkg/config/config_template.yaml

```diff
@@ -1480,6 +1480,13 @@ api_key:
   ## may not be marked by the Agent as top-level spans.
   # peer_service_aggregation: false
 
+  ## @param peer_tags - list of strings - optional
+  ## @env DD_APM_PEER_TAGS - list of strings - optional
+  ## Enables additional tags related to peer.service. Will not be used unless peer_service_aggregation is also set.
```

Comment:
> Enables additional tags related to peer.service.

Do you think customers will have enough context to know what this means? I personally doubt that most would. Is there a link to some public docs that describe this that you can link here?

Reply:
Discussed offline; public docs are coming and I will mark `[BETA]` in the description.